scraper.py

```diff
@@ -121,6 +130,32 @@ def get_posts(candidates):
             posts[candidate['post_id']]['known_candidates'] += 1
     return posts
 
+
+def get_sopn_date(result):
+    slug = result['election_id']
+    territory = result['organisation']['territory_code']
+
+    country = {
+        "ENG": Country.ENGLAND,
+        "WLS": Country.WALES,
+        "SCT": Country.SCOTLAND,
+        "NIR": Country.NORTHERN_IRELAND,
+    }
+
+    if slug.startswith("local") and territory not in country:
```

Comment:
Should this be `local` and `parl`? I can't remember if NI and Scotland have different rules for Parlimentary election or only local. You've probably done more research on this than me..

Reply:
The number of days is the same but the difference is whether there's a country-specific public holiday.

This should fold `parl` into the check too, which means it's missing this logic in WCIVF too - more argument for pulling this logic into the `for_id` function rather than just bailing if it's a local or parl.